packages/cdk/lib/roles/policies/nextflow-submit-job-batch-policy.ts

```diff
@@ -0,0 +1,17 @@
+import * as iam from "monocdk/aws-iam";
+import { NextflowAdapterRoleProps } from "../nextflow-adapter-role";
+
+export class NextflowSubmitJobBatchPolicy extends iam.PolicyDocument {
+  constructor(props: NextflowAdapterRoleProps) {
```

Comment:
NIT: This class should have it's own props and NextflowAdapterRoleProps should extend this classes props instead of the other way around

Reply:
Adjusted 👍